### ISSUE TITLE
Nodemon crashes on exactly every other file change.

### ISSUE THREAD

**Issue by mikestopcontinues** [remy/nodemon#1188]
### Expected behaviour

When I change a file, it should reload.

### Actual behaviour

Every exactly other time I change a file, I get this error:

```
[nodemon] app crashed - waiting for file changes before starting...
```

On the following change, it reloads. (Note: change could be as basic as a `touch`.)

I tried increasing the delay, but it did nothing.

```
--------------
node: v9.2.0
nodemon: 1.13.2
command: /usr/local/bin/node /Users/msc/Code/mikestopcontinues.com/node_modules/.bin/nodemon --watch generator --dump generator/generator.js
cwd: /Users/msc/Code/mikestopcontinues.com
OS: darwin x64
--------------
{ run: false,
  system: { cwd: '/Users/msc/Code/mikestopcontinues.com' },
  required: false,
  dirs: [ '/Users/msc/Code/mikestopcontinues.com/generator' ],
  timeout: 1000,
  options:
   { watch: [ 'generator', re: /generator/ ],
     dump: true,
     ignore:
      [ '.git',
        '.nyc_output',
        '.sass-cache',
        'bower_components',
        'coverage',
        'node_modules',
        re: /\.git|\.nyc_output|\.sass\-cache|bower_components|coverage|node_modules/ ],
     ignoreRoot:
      [ '.git',
        '.nyc_output',
        '.sass-cache',
        'bower_components',
        'coverage',
        'node_modules' ],
     restartable: 'rs',
     colours: true,
     execMap: { py: 'python', rb: 'ruby' },
     stdin: true,
     runOnChangeOnly: false,
     verbose: false,
     signal: 'SIGUSR2',
     stdout: true,
     watchOptions: {},
     execOptions:
      { script: 'generator/generator.js',
        exec: 'node',
        args: [],
        scriptPosition: 0,
        nodeArgs: undefined,
        execArgs: [],
        ext: '',
        env: {} },
     monitor:
      [ '/Users/msc/Code/mikestopcontinues.com/generator/**/*',
        '!/Users/msc/Code/mikestopcontinues.com/.git/**/*',
        '!.nyc_output',
        '!.sass-cache',
        '!bower_components',
        '!coverage',
        '!/Users/msc/Code/mikestopcontinues.com/node_modules/**/*' ] },
  load: [Function],
  reset: [Function: reset],
  lastStarted: 0,
  loaded: [ '/Users/msc/Code/mikestopcontinues.com/package.json' ],
  watchInterval: null,
  signal: 'SIGUSR2',
  command:
   { raw: { executable: 'node', args: [ 'generator/generator.js' ] },
     string: 'node generator/generator.js' } }
--------------
```

**Comment by mikestopcontinues** [comment]
The bug even occurs every other manual refresh via `rs`.

**Comment by remy** [comment]
Please upgrade to latest. Fix has already been deployed. Ta!

**Comment by mikestopcontinues** [comment]
Thanks!

**Comment by Noerdsteil** [comment]
Still having this issue. 
using nodemon 1.14.12, Webstorm on Mac OX High Sierra. 
I am on node 9.5.0 
In my app I am using recluster

Could you please provide further hints, how to solve this issue

**Comment by remy** [comment]
@Noerdsteil This is a closed issue, unless this thread helps you, you'll need to raise a new issue entirely.
